From ed08bfa09665e69f5a781961b87eed5450ddb2c7 Mon Sep 17 00:00:00 2001
From: Kai Liu <kai.liu@suse.com>
Date: Mon, 27 Jun 2022 19:00:19 +0800
Subject: [PATCH] Revert "Linux 5.10.126"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 9cae50bdfafa0ce87eb2693401efeae2cd30b417.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 57434487c2b4..da5b28931e5c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 126
+SUBLEVEL = 125
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.1

